### PR TITLE
Validate `composite_tree`

### PR DESCRIPTION
**Pull request by uvsmtid-gs**
*   Implement multiple `CompositeForestValidator*`.
*   Implement validation for missing `prop_name`-s used in `search_control` on server start.
*   Derive automatically for `compoiste_forest`:
    *   `next_interp.jump_path`
    *   `next_interp.plugin_instance_id`
*   Rename: `composite_tree` to `composite_forest`.
*   Rename: `*_func` to `func_id_*`.
*   Minor: disable loading cluster in `ServiceLoader` until it is needed.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -35,8 +35,9 @@ def list_dir(
 
 setuptools.setup(
     name = "argrelay",
+    # TODO: extract version from `_version.py` file:
     # See `docs/dev_notes/version_format.md`:
-    version = "0.7.8",
+    version = "0.7.9",
```
Comment (uvsmtid) [inline setup.py:40]: Finalize before merge.

File: src/argrelay/_version.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+# Implements this:
+# https://stackoverflow.com/a/7071358/441652
+__version__ = "3.6.5"
```
Comment (uvsmtid) [inline src/argrelay/_version.py:4]: Implement or remove.

File: src/argrelay/relay_server/CustomFlaskApp.py [hunk 3]
```diff
@@ -17,6 +17,7 @@
 from argrelay.schema_config_plugin.PluginConfigSchema import plugin_config_desc
 from argrelay.server_spec.const_int import API_SPEC_PATH, API_DOCS_PATH, ARGRELAY_GUI_PATH
 
+# TODO: Search for pkg_resources and replace version extraction to `argrelay.__version__`:
```
Comment (uvsmtid) [inline src/argrelay/relay_server/CustomFlaskApp.py:20]: Implement or remove.

Comment (uvsmtid-gs) [reply src/argrelay/relay_server/CustomFlaskApp.py]: Removed.

File: src/argrelay/_version.py (hunk 2 above)
Comment (uvsmtid-gs) [reply src/argrelay/_version.py]: Removed.

File: docs/task_refs/TODO_99_87_25_42.next_incompatible_changes.md [hunk 4]
```diff
@@ -27,7 +27,7 @@ TODO:
 *   TODO: Move `gui_banner_config` under `ConfiguratorDefault`
 
 *   TODO: Rename all `func_id` to be prefixed with `func_id`, for example, `func_id_query_enum_items` -> `func_id_query_enum_items`.
-*   TODO: Rename: prefix all `func_id` with `func_id`, for example: `goto_service_func` -> `func_id_goto_service`.
+*   TODO: Rename: prefix all `func_id` with `func_id`, for example: `func_id_goto_service` -> `func_id_goto_service`.
```
Comment (uvsmtid) [inline docs/task_refs/TODO_99_87_25_42.next_incompatible_changes.md:30]: Clean up.

File: setup.py (hunk 1 above)
Comment (uvsmtid-gs) [reply setup.py:39]: Let it be 0.7.9

**Review by uvsmtid — APPROVED** [review]

**Pull request merged**